core/roboconf-agent-monitoring/src/main/java/net/roboconf/agent/monitoring/internal/MonitoringTask.java

```diff
@@ -138,14 +155,14 @@ public void run() {
 				continue;
 
 			if( s.toLowerCase().startsWith( RULE_BEGINNING )) {
-				sections.add( sb.toString());
+				if(sb.length() > 0) sections.add( sb.toString());
```

Comment:
The check was wrong - fixed now (not to include empty sections). Not so important (empty sections are filtered out when present...) but right !